### PR TITLE
feat(role-based-access): Permissions table display

### PR DESCRIPTION
**Pull request by liyiy**
## Changes
-  table display for roles permissions on a specific feature flag
Screen Shot 2022-11-30 at 11 06 38 PM

- setting permissions on a new feature flag
Screen Shot 2022-11-30 at 11 06 55 PM

- option to add any team member/yourself to a role so it's not restricted to non-admin members: 
ex: I am trying to add Eric to team experiments but he doesn't show up since he's an admin
Screen Shot 2022-11-30 at 11 05 46 PM

- split up role access into a separate tab in organization settings
Screen Shot 2022-11-30 at 11 23 36 PM

https://www.figma.com/file/UBPG5Wjpe0EbfABdlCavt1/Feature-Flag-Permissions-Mocks?node-id=1%3A798&t=uVGYqUzD3ENBZuud-0

### Todo
- Implement a better way to display roles information, for now we'll just show all the members and feature flags it's connected to for testing. When it's a "edit" role, it says 'All flags' instead

Screen Shot 2022-11-30 at 11 36 02 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by EDsCODE — COMMENTED** [review]
On this page in the feature flag permissions tab, I think we should add a link back to the org settings in case people see this and want to edit the defaults

Screen Shot 2022-12-01 at 11 18 20 AM

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx [hunk 1]
```diff
@@ -241,9 +241,19 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                 </Field>
                             </Col>
                             <Col span={12}>
-                                <LemonButton type="secondary" onClick={() => setModalOpen(true)} className="mb-4">
-                                    Set permissions
-                                </LemonButton>
+                                <Card title="Permissions" className="mb-4">
```
[the remark this answers is not in the document]
Comment (liyiy) [reply frontend/src/scenes/feature-flags/FeatureFlag.tsx:244]: nope! good catch

**Comment by liyiy** [comment]
> On this page in the feature flag permissions tab, I think we should add a link back to the org settings in case people see this and want to edit the defaults
> 
> Screen Shot 2022-12-01 at 11 18 20 AM

Makes sense, I'll add this, anything else blocking?

**Comment by EDsCODE** [comment]
Nope, nothing else blocking

**Comment by liyiy** [comment]
Everything addressed

**Pull request merged**